modules/siddhi-core/src/main/java/io/siddhi/core/table/holder/IndexEventHolder.java

```diff
@@ -61,12 +61,12 @@
     private static final Logger log = Logger.getLogger(IndexEventHolder.class);
     private static final long serialVersionUID = 1272291743721603253L;
     private static final float FULL_SNAPSHOT_THRESHOLD = 2.1f;
-    private final Map<Object, StreamEvent> primaryKeyData;
-    private final Map<String, TreeMap<Object, Set<StreamEvent>>> indexData;
+    final Map<Object, StreamEvent> primaryKeyData;
```

Comment:
why some objects have made non-private, is there are subclass usage make them protected.